pkg/crc/version/version.go

```diff
@@ -87,7 +87,12 @@ func GetCRCLatestVersionFromMirror() (*CrcReleaseInfo, error) {
 	client := &http.Client{
 		Timeout: 5 * time.Second,
 	}
-	response, err := client.Get(releaseInfoLink)
+	req, err := http.NewRequest(http.MethodGet, releaseInfoLink, nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("User-Agent", fmt.Sprintf("crc/%s", crcVersion))
```

Comment:
@guillaumerose this is only added to understand for which version the request made?

Reply:
yes, it is also a good practice to specify a proper user agent. I should also add the platform.